src/Cms/Responder.php

```diff
@@ -99,13 +116,71 @@ public function body(string $body = null)
     public function cache(?bool $cache = null)
     {
         if ($cache === null) {
+            // never ever cache private responses
```

Comment:
technically, inside this if statement, this would rather be "never ever return cache for private responses". question is if the new if statement shouldn't be at the top of the method to prevent getting as well as setting cache

Reply:
`Response::$cache` is the flag that controls whether the current response can be cached, not whether a cached response can be returned.

This `if` only overrides the manually set value if `isPrivate()` is true. Otherwise the manual value "shines through". So I don't think we should limit the setter.